### PR TITLE
[stable/kong] major breaking change: dbless and KIC by-default

### PR DESCRIPTION
**Pull request by hbagdi**
Also, a changelog section has been started that should be updated for
each and every release.

Signed-off-by: Harry Bagdi <harrybagdi@gmail.com>

#### Special notes for your reviewer:

We are starting a new Changelog section with this release to easily communicate changes with each release to end-users. The changelog will be available on the web at the following location:
https://hub.helm.sh/charts/stable/kong

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by shashiranjan84** [comment]
@hbagdi  https://github.com/helm/charts/blob/89f702edfdb0ef90326b4e2f79f3d2ddcc41f408/stable/kong/README.md#kong-specific-parameters needs to updated. It is still saying that `postgresql` is default database option

**Comment by hbagdi** [comment]
@shashiranjan84 fixed.

**Comment by shashiranjan84** [comment]
> @shashiranjan84 fixed.

I don't see any change, it still sayin PostgreSQL is default

`Kong has a choice of either Postgres or Cassandra as a backend datatstore. This chart allows you to choose either of them with the env.database parameter. Postgres is chosen by default.`

**Comment by hbagdi** [comment]
@shashiranjan84 My bad. Fixed now.

File: stable/kong/README.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ for the service definition to work properly.
 
 Kong has a choice of either Postgres or Cassandra as a backend datatstore.
 This chart allows you to choose either of them with the `env.database`
-parameter. Postgres is chosen by default.
+parameter. The default is to use Kong without a database.
```
Comment (shashiranjan84) [inline stable/kong/README.md]: This whole paragraph needs to be updated to make sense.

**Comment by hbagdi** [comment]
Let's try this one more time now. Pushed an update.

**Review by shashiranjan84 — APPROVED** [review]
/lgtm

**Pull request merged**